test_crosscompile.sh

```diff
@@ -49,12 +49,12 @@ echo2 "Build for linux."
 echo2 "  - linux/riscv"
 docker build -t creack-pty-test -f Dockerfile.riscv .
 
-# Golang dropped support for darwin 32bits since go1.15. Make sure the lib still compile with go1.14 on those archs.
+# Golang dropped support for darwin 32bits since go1.15. Make sure the lib still compile with go1.18.2 on those archs.
```

Comment:
FYI, I reverted the changes to that file. This point is to make sure it compiles with go < 1.15 as 1.15 dropped support for darwin/32bits.